docs/list_of_apis.jl

```diff
@@ -12,7 +12,11 @@ apis = [
     ],
     "Ocean" =>
         ["Hydrostatic Boussinesq" => "APIs/Ocean/HydrostaticBoussinesq.md"],
-    "Land" => [],
+    "Land" => [
+        "LandModel" => "APIs/Land/LandModel.md",
+        "SoilWaterParameterizations" =>
```

Comment:
```suggestion
        "Soil Water Parameterizations" =>
```
This is the title of the navigation page, so it should read as plain text and not code.